src/source/Ice/IceAgent.c

```diff
@@ -695,6 +715,145 @@ STATUS iceAgentShutdown(PIceAgent pIceAgent)
     return retStatus;
 }
 
+STATUS iceAgentRestart(PIceAgent pIceAgent, PCHAR localIceUfrag, PCHAR localIcePwd)
```

Comment:
I think iceAgentReset and iceAgentRestart has similar meanings, but people may be more familiar with ice restart than ice reset since its formally defined.

Reply:
I think reset just means clearing the settings, restart is a reset + start. But, whatever you feel makes sense :)